desktop/cmp/input/RadioInput.ts

```diff
@@ -99,6 +99,7 @@ const cmp = hoistCmp.factory<RadioInputModel>(({model, className, ...props}, ref
             label: opt.label,
             value: opt.value,
             className: 'xh-radio-input-option',
+            'data-testid': getTestId(props.testId, `${opt.value}`),
```

Comment:
If we are generating a sub-testId here for radio inputs based on value, we should be doing it for analogous controls like `ButtonGroupInput` - but are we sure this is reliable and will work for all accepted values?  

Also - we should have the unmodified testId installed on the outer `radioGroup` unless there's some reason it can't take it.

Reply:
Would label be a viable option? It is typed as a string but is optional

Reply:
I guess we just want to make sure we never crash or fail.  Much less important if the user ends up with an unusable test-id of course for a non-primitive or unusual value.